chefignore

```diff
@@ -97,7 +97,6 @@ Policyfile.lock.json
 
 # Cookbooks #
 #############
-CHANGELOG*
```

Comment:
This is already removed on master. @xorimabot nuked that already

Reply:
weird my fork ended up having it.